pkg/mdformatter/mdformatter.go

```diff
@@ -240,6 +280,12 @@ func format(ctx context.Context, logger log.Logger, files []string, diffs *Diffs
 			return file.Truncate(int64(n))
 		}())
 	}
+	if spin != nil {
+		err := spin.Stop()
+		if err != nil {
+			errs.Add(err)
+		}
+	}
```

Comment:
```suggestion
	if spin != nil {
	  errs.Add(spin.Stop())
	}
```
Add checks error already

Reply:
Sure!